gbif/pipelines/ingest-gbif-beam/src/main/resources/elasticsearch/es-occurrence-schema.json

```diff
@@ -203,6 +203,7 @@
       "gadm": {
         "type": "object",
         "properties": {
+          "gid": {"type": "keyword"},
```

Comment:
Looks like we use plural naming convention in our schema. `gids`  ?

Reply:
I wasn't sure, as we have `taxonKey` which is singular.